### DISCUSSION TITLE
¿Cómo puedo configurar Polygon en Metamask?

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/provida#8] [Q&A] [answered]
¿Cómo puedo configurar Polygon en Metamask?

**Comment by ruth-ga** [comment, marked as answer]
En  estos momentos hay varios videos/tutoriales que ayudan a configurar la red Polygon a nuestra Billetera Metamask .  Básicamente es 
1,-tener ya nuestra Billetera Metamask  activa 
2,-CLIKAR En el recuadro superior donde figura "red principal Ethereum", deslizarse hacia abajo  hasta "Agregar Red"
3.- rellenar el formulario con estos datos:
Nombre: Matic Mainnet
Url: https://rpc-mainnet.maticvigil.com/
ID: 137
SIMBOLO : MATIC
Block explorer url: https://explorer.matic.network/
y GUARDAR.
